sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -623,9 +625,30 @@ class SdkSynchronizer private constructor(
 
     override suspend fun validateConsensusBranch(): ConsensusMatchType {
         val serverBranchId = tryNull { processor.downloader.getServerInfo()?.consensusBranchId }
-        val sdkBranchId = tryNull {
-            (txManager as OutboundTransactionManagerImpl).encoder.getConsensusBranchId()
+
+        val currentChainTip = when (
```

Comment:
Note that this rather fetches the latest chain tip from the network than using the cached one, as I'm not sure what the common use-case for this API is and, thus if the cached `networkHeight` is prepared.